### PR TITLE
PLFM-7235: Fix for CVE-2021-33813

### PR DESCRIPTION
**Pull request by harveyx22**

### CODE REVIEW

**Review by marcomarasca — CHANGES_REQUESTED** [review]
While it seems to work just updating to jdom2, from the migration documentation (https://github.com/hunterhacker/jdom/wiki/JDOM2-Migration-Issues) it looks like this is not enough. In particular they mention that some methods were deprecated in jdom2 and need to be changed. An example is the SAXBuilder.setValidation. Note that even though jdom was had the CVE our code is not affected (we are not using jdom for user input) so it is not strictly necessary to upgrade (but it'd be a good thing to be on the latest).

**Comment by harveyx22** [comment]
Deprecated method setValidation replaced per http://www.jdom.org/docs/apidocs/org/jdom2/input/SAXBuilder.html#setValidation-boolean-

File: pom.xml [hunk 1]
```diff
@@ -635,7 +635,7 @@
 			<dependency>
 				<groupId>org.sagebionetworks</groupId>
 				<artifactId>database-semaphore</artifactId>
-				<version>3.0.6</version>
+				<version>3.0.5</version>
```
Comment (marcomarasca) [inline pom.xml]: Why are we reverting all the versions?

**Review by marcomarasca — CHANGES_REQUESTED** [review]
Looks like you reverted the database-semaphore, aws-utilities and simple-http-client version

**Review by marcomarasca — APPROVED** [review]

**Pull request merged**